src/components/TravelBlog/todo/todo.module.css

```diff
@@ -0,0 +1,45 @@
+.todoList {
+  text-align: center;
+}
+
+.sn {
+  color: #fff;
+  background: #5ebfbf;
+  width: 64px;
+  margin: 10px auto;
+  border-radius: 50%;
+  padding: 22px 0;
+  font-weight: bold;
+}
+
+.todoHeader {
+  font-size: 18px;
+  color: #555555;
+  font-weight: 600;
+  margin: 10px 0;
+}
+
+.todoBody {
+  font-size: 14px;
+  color: #888888;
```

Comment:
Would be nice to have a variable css file so we can store there common colors font sizes and all the common styles properties values